src/ducks/drafts.js

```diff
@@ -106,6 +111,24 @@ export const deleteDraft = (directory, filename) => dispatch => {
     );
 };
 
+export const publishDraft = (directory, filename) => (dispatch, getState) => {
+  const metadata = getState().metadata.metadata;
+  let { raw_content, title } = metadata;
```

Comment:
This isn't changing. We can make it `const`?

Also, `title` is not used?

Reply:
Oops.. Good catch. :+1: